pulp_2to3_migration/app/migration.py

```diff
@@ -191,6 +193,7 @@ def complex_repo_migration(plugin, pulp3_repo_setup, repo_name):
             pass
     for pulp2_repo_info in repo_versions_setup:
         try:
+            # TODO: should we check the is_migrated=True?
             pulp2_repo = Pulp2Repository.objects.get(
```

Comment:
same here, is_migrated=False

Reply:
and one line 216 is_migrated=True
`migrated_repo = Pulp2Repository.objects.get(pulp2_repo_id=repo_id, is_migrated=True, not_in_plan=False)`